tox.ini

```diff
@@ -31,10 +34,14 @@ deps =
     django42: Django>=4.2.a1,<4.3
     django50: Django>=5.0a1,<5.1
     django51: Django>=5.1,<5.2
```

Comment:
We can probably remove all these dependency definitions since this PR bumps the lower bounds of django to 5.2.